### ISSUE TITLE
Warning when build with Xcode 8 GM

### ISSUE THREAD

**Issue by leavez** [ReactiveCocoa/ReactiveCocoa#3183]
When compiling with Xcode 8 GM,  the `RACObserve` macro in the latest release generate a warning  at 

```
_Pragma("clang diagnostic ignored \"-Wreceiver-is-weak\"") \
```

```
Unknown warning group '-Wreceiver-is-weak', ignored
```

It appears at every time I use a `RACObserve`, which in my project it produces 100+ warnings.

**Comment by SkyHarute** [comment]
When I use OC on Xcode 8:
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

**Comment by SkyHarute** [comment]
I should use 2.5...

**Comment by jinke5245** [comment]
@SkyHarute 
`Target` -> `Build Settings` -> `Use Legacy Swift Language Version` set **YES**

**Comment by SkyHarute** [comment]
@jinke5245 兄弟这是swift才有的,我使用OC写的

**Comment by yageek** [comment]
I think this warning is not used since XCode 7 ->  http://lists.llvm.org/pipermail/cfe-commits/Week-of-Mon-20160418/156202.html

**Comment by jinke5245** [comment]
@SkyHarute 我也是oc项目，我通过cocoapods导入的reactivecocoa。我通过上面的办法，修改了pods工程里reactivecocoa的target解决此问题。

**Comment by mdiep** [comment]
This was fixed by #3186.
